### DISCUSSION TITLE
How to use a theme file in eza installed with winget?

### DISCUSSION THREAD

**Discussion by erikgiovani** [eza-community/eza#1168] [Q&A, Howto, Help] [answered]
I try config the env EZA_CONFIG_DIR with the theme.yml path but it doesn´t work

**Comment by rahaaatul** [comment]
Did you find the answer?

**Reply by erikgiovani** [reply]
No

**Comment by AeliusSaionji** [comment]
The default location appears to be `%APPDATA%\eza`.
The environment variable method does work for me, though.

**Reply by rahaaatul** [reply]
Can you elaborate how did you make it work?

**Reply by AeliusSaionji** [reply, marked as answer]
The exact command I ran in powershell 7 is
```powershell
Set-Item -Path Env:\EZA_CONFIG_DIR 'C:\Users\aelius\.config\eza'
```

With the theme.yml in said folder.

You can use any method to set an environment variable though.

**Reply by erikgiovani** [reply]
Thanks for the helpfull @AeliusSaionji 👍

**Reply by rahaaatul** [reply]
**Edit**: Solved! Had to use `absolute path` to use the theme.

> You can use any method to set an environment variable though.

I've set env variable like this: `$env:EZA_CONFIG_DIR='$env:userprofile\.config\eza'` in powershell config file. and catppuccin theme file as theme.yml in `eza` folder. But my icons are not colored at all, see this:

![image](https://github.com/user-attachments/assets/39e4ba7d-a78c-48dc-8508-f62416c33528)
